### PR TITLE
Alert component added which dimisses on clicking the button

### LINKED ISSUE #12: Feature request: create a alert component #11
### Description

Create Alerts available for any length of text, as well as an optional dismiss button. For proper styling, use one of the required contextual classes (e.g., `.alert-success`).

### PR DESCRIPTION
**Pull request by Nimanita**
closes #12 

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Is your feature request related to a problem? Please describe.**

**Does this PR introduce a breaking change?** (check one)

- [x] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**Describe the solution you'd like**

**Additional context** (others informations)

### CODE REVIEW

**Comment by Nimanita** [comment]
I have added _alert.css  link in alert.html .After adding the file in the server.you can delete the line.Attaching the screenshot

File: src/_alert.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ 
+    transform:scale(1.03);
+    opacity:3;
+    
+}
+
+
+
+
+.danger
```
Comment (rodgeraraujo) [inline src/_alert.css:52]: For color use the default please,  you see the palette [colors](https://github.com/rodgeraraujo/minus.css/blob/dev/src/_color.css). Use the vars to get color dynamic, like `var(--color-secondary)`.

File: src/_alert.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
+.fieldset
+{
+    width:100%;
```
Comment (rodgeraraujo) [inline src/_alert.css:3]: For spacing use the `rem` pattern as opposed to`%`, you can find more details about the spacing [here](https://github.com/rodgeraraujo/minus.css/blob/dev/src/_spacing.css).

File: test/alert.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>alert - Elements test</title>
+    <link rel="stylesheet" href="https://unpkg.com/minus.css" />
+    <link rel="stylesheet" href="../src/_alert.css">
```
Comment (rodgeraraujo) [inline test/alert.html:9]: Here, you use the `dist` folder to test your modifications.

**Review by rodgeraraujo — COMMENTED** [review]
Please correct the pending issues, I appreciate the contribution.

**Comment by Nimanita** [comment]
I have made the changes regarding spacing and colors but I didn't understand how to check using dist folder?It will be of great help if you could elaborate it.
Thank you

**Comment by Nimanita** [comment]
I am not sure whether my method to check with dist folder is correct .I had put all my _alert.css code in dist/minus.css file .It was successful.Attaching the screenshot

**Comment by Nimanita** [comment]
please add `hacktoberfest-accepted` label to my pull request.So that it will get counted towards hacktoberfest.

Thank you.

**Comment by rodgeraraujo** [comment]
@Nimanita Hi, sorry for the delay in responding. That's right, I will merge as soon as possible and add as `hacktoberfest-accepted`, thank you very much for contributing.

**Pull request merged**